hw/ip/flash_ctrl/dv/xfile

```diff
@@ -0,0 +1 @@
+SCOPE tb.dut F
```

Comment:
Yes,for xcelium

Reply:
why the other blocks don't need to add this file? if it's needed for xcelium, we should add this in a common place like dv/tools

Reply:
Actually this file is mandatory for all tests. Simulation is done via Candence Xcelium. I have experienced infinite loop using Xcelium with enabled XPROP switch (default value) which i do not see using VCS. Solution for this was adding switch -xfile and xfile itself in which i put SCOPE tb.dut F. Xfile switch overrides xprop "xmelab: *W,XPOPT: If both -XFILE and -XPROP options are present, -XPROP will be ignored." . I already asked that on slack and provided branch for tryout but no response.  I can add it to dv/tools.

Reply:
> Actually this file is mandatory for all tests. Simulation is done via Candence Xcelium. I have experienced infinite loop using Xcelium with enabled XPROP switch (default value) which i do not see using VCS. Solution for this was adding switch -xfile and xfile itself in which i put SCOPE tb.dut F. Xfile switch overrides xprop "xmelab: *W,XPOPT: If both -XFILE and -XPROP options are present, -XPROP will be ignored." . I already asked that on slack and provided branch for tryout but no response. I have added it to dv/tools.

we enable xprop for both VCS and Xcelium, which is required in our [methodology](https://github.com/lowRISC/opentitan/blob/master/doc/ug/dv_methodology/index.md#x-propagation-xprop). we can check in this file temporary, but please file a Github issue and try to dig into it later. Maybe there is some unknown propagated, which causes a infinite loop. If so, it could be a really issue.

Reply:
https://github.com/lowRISC/opentitan/issues/9661. I have put you and the rest of DV team, so please re-assign to right person.